docs/rules/a11y-no-title-attribute.md

```diff
@@ -0,0 +1,43 @@
+# Guards against developers using the title attribute (`github/a11y-no-title-attribute`)
+
+<!-- end auto-generated rule header -->
+
+The title attribute is strongly discouraged. The only exception is on an `<iframe>` element. It is hardly useful and cannot be accessed by multiple groups of users including keyboard-only users and mobile users.
```

Comment:
Most of this is directly from the [rails rule](https://github.com/github/erblint-github/blob/main/docs/rules/accessibility/no-title-attribute.md). With the exception of the last sentence.